src/test/java/tests/FilterTest.java

```diff
@@ -0,0 +1,39 @@
+package tests;
+
+import indices.*;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.ApplePage;
+import pages.DominoPage;
+import pages.MainPage;
+import pages.SmartphonePage;
+import testData.Catalogue;
+
+
+public class FilterTest extends BaseTest {
+
+    @Test(description = "Positive test to choose Apple manufacturer in the filter")
+    public void chooseManufacturerTest() {
+        ApplePage applePage = new MainPage()
+                .openPage()
+                .clickOnCatalogue()
+                .getIconOption(Icons.ELECTRONICS)
+                .getLeftDropdown(Icons.ELECTRONICS, Electronics.MOBILE_PHONES)
+                .getMiddleDropdown(SmartphonePage.class, Pages.SMARTPHONE, Icons.ELECTRONICS, PhoneDevices.SMARTPHONES)
+                .chooseManufacturer(ApplePage.class, Pages.APPLE, Catalogue.BRAND)
```

Comment:
maybe it's worth renaming the method as you are choosing not only Manufacturer there?

Reply:
we can choose only Manufacturer